packages/app-aco/src/components/AdvancedSearch/GraphQLInputMapper.ts

```diff
@@ -9,10 +9,22 @@ export class GraphQLInputMapper {
                         const { field, condition, value } = filter;
                         const key = `${field}${condition}`.trim();
 
-                        return { [key]: value };
+                        return { [key]: this.convertToBooleanOrString(value) };
                     })
                 };
             })
         };
     }
+
+    private static convertToBooleanOrString(value: string | boolean): string | boolean {
+        if (value === "true") {
+            return true;
+        }
+
+        if (value === "false") {
+            return false;
+        }
+
+        return value || "";
```

Comment:
If the value is false (boolean) you will get empty string as a result.

Reply:
Thanks @brunozoric, for pointing this out. I changed it into the nullish coalescing operator.